src/systems/fem_system.C

```diff
@@ -246,7 +246,7 @@ void assemble_unconstrained_element_system(const FEMSystem & _sys,
     }
 }
 
-void add_element_system(const FEMSystem & _sys,
+void add_element_system(FEMSystem & _sys,
```

Comment:
So there's another nice benefit of accessors: it makes it more obvious when our "const" specifiers are actually lies.